inc/instance-config.php

```diff
@@ -82,9 +96,12 @@
 $config['cookies']['mod'] = 'mod';
 $config['cookies']['salt'] = 'MGYwNjhlNjU5Y2QxNWU3YjQ3MzQ1Yj';
 
-$config['flood_time'] = 30;
-$config['flood_time_ip'] = 60;
-$config['flood_time_same'] = 60;
+
+$config['flood_cache'] = 60 * 15; // 15 minutes
+$config['flood_time_any'] = 40; // in seconds
+$config['flood_time'] = 0;
```

Comment:
these need to be changed back to 30 or 60, the only new value added here is flood_time_any